src/components/Atoms/Text/Text.tsx

```diff
@@ -1,13 +1,12 @@
 import styled from 'styled-components';
-import { UIprops } from '../../../utils/ui-props';
 
-import { generalProps, GeneralProps } from '../../../theme/props';
+import {
+	generalProps, GeneralProps,
+} from '../../../theme/props';
 
-export type TText = GeneralProps;
+export type TText = GeneralProps
 
-export const Text = styled.span.withConfig({
```

Comment:
@Ivana-Jovic ajde probaj sad i vrati withConfig, posto to nije razlog gresaka bio